megamek/src/megamek/client/ui/swing/dialog/imageChooser/AbstractIconChooser.java

```diff
@@ -252,7 +165,8 @@ private void updateSearch(String contents) {
     }
 
     /** Returns the selected AbstractIcon. May be null. */
-    public @Nullable AbstractIcon getSelectedItem() {
+    public @Nullable
+    AbstractIcon getSelectedItem() {
```

Comment:
Why the line split?

Reply:
Annoying IDE automatic format when copying/pasting that I cannot find how to turn off. Fixed.